cloud/migrate-mst-cloud/entire-database.md

```diff
@@ -0,0 +1,146 @@
+# Migrate entire database at once
+
+Migrate smaller databases by dumping and restoring the entire database at once.
+
+<highlight type="warning"> 
+Depending on your database size and  network speed,
+migration may take several hours. You can continue reading from your old
+database during this time, though performance may be slower. Writes made to your
+old database during migration might not be copied to Timescale Cloud. Plan your migration accordingly to avoid data loss. 
+</highlight>
+
+## Prerequisites
+
+- Check that the PostgreSQL utilities [`pg_dump`][pg_dump] and
+  [`pg_restore`][pg_restore] are installed:
+
+  ```bash
+  pg_dump --version
+  pg_restore --version
+  ```
+
+- Create a new empty database in Timescale Cloud. For instructions on installing
+  Timescale Cloud, see [Install Timescale Cloud][install-timescale-cloud].
+
+- Check that you're running the same major version of PostgreSQL on both Managed
+  Service for TimescaleDB and Timescale Cloud. To upgrade your version, refer to
+  the section on [upgrading PostgreSQL][upgrading-postgresql].
+
+- Check that you're running the same major version of TimescaleDB on both
+  Managed Service for TimescaleDB and Timescale Cloud. To upgrade your version,
+  refer to the section on [upgrading TimescaleDB][upgrading-timescaledb].
```

Comment:
Upgrades for managed services are done from the Console directly.
For MST: https://kb-managed.timescale.com/en/articles/5368016-perform-a-postgresql-major-version-upgrade
For Cloud: Currently users can't upgrade/downgrade versions

Reply:
same applies for Strategy 2

Reply:
Thanks! That link is for _PostgreSQL_ upgrades on MST, right? (It doesn't cover TimescaleDB upgrades) Is the current link for TimescaleDB upgrades correct? 

(Also, does this mean we need to update the upgrading guides in our current docs? There's no information aside from the self-hosted update instructions I originally linked.)

Reply:
the docs for upgrading TimescaleDB should be fine
Cloud/MST users can do that from within the database

regarding your section question, I think someone from the docs team needs to answer this. I think there are a lot of articles in the MST knowledge base that aren't covered in our docs, and I'm not sure if we want to have them in our docs.

Reply:
ok I'll open a separate issue to discuss that with the docs team and mark this resolved then